lib/pxutil.js

```diff
@@ -323,5 +328,7 @@ module.exports = {
     generateHMAC,
     isReqInMonitorMode,
     getTokenObject,
-    getGraphqlData
+    getGraphqlData,
+    isEmailAddress,
+    _hashString
```

Comment:
The underscore here suggests that the function should be private. At the moment, the function `sha256()` is already be exported so it's possible to just use that.